### ISSUE TITLE
SSH Use of __getattr__ vs __call__

### ISSUE THREAD

**Issue by zachriggle** [Gallopsled/pwntools#220]
In retrospect, this should have always been `__call__`.  Not sure why I used `__getattr__` in the first place.

Proposed change:

```python
shell = ssh(host='bandit.labs.overthewire.org',user='bandit0',password='bandit0')

# Old
print shell['ls']

# New
print shell('ls')
```

**Comment by IdolfHatler** [comment]
So. What do we do about this?

**Comment by zachriggle** [comment]
This was fixed in bb3d98f which was merged in with #249
